test/jdk/java/awt/im/JTextFieldTest.java

```diff
@@ -23,43 +23,51 @@
 
 /*
  * @test
- * @bug  4226191
+ * @bug 4226191
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
  * @summary  Verify that Lightweight text components (like swing JTextField)
  *           work correctly with IM when there is an uneditable peered
  *           TextField/TextArea in the same parent Frame
- * @author xueming.shen@eng
- * @run applet/manual=yesno JTextFieldTest.html
+ * @run main JTextFieldTest
  */
 
-import java.awt.*;
-import java.awt.event.*;
-import java.applet.*;
-import javax.swing.*;
+import java.awt.FlowLayout;
+import java.awt.TextField;
 
-public class JTextFieldTest extends Applet implements ActionListener {
+import javax.swing.JFrame;
+import javax.swing.JTextField;
 
-    TextField  tf1;
-    JTextField tf2;
+public class JTextFieldTest {
+    private static JFrame frame;
```

Comment:
can be moved to local scope in `createAndShowGUI` method.

Reply:
Fixed.